opentelemetry-sdk/src/metrics/meter_provider.rs

```diff
@@ -174,6 +174,10 @@ impl MeterProvider for SdkMeterProvider {
 
     fn meter_with_scope(&self, scope: InstrumentationScope) -> Meter {
         if self.inner.is_shutdown.load(Ordering::Relaxed) {
+            otel_debug!(
+                name: "MeterProvider.NoOpMeterReturned",
```

Comment:
Since this event is used at two different places, it would be nice to add a message field to distinguish between the two. Message for this one could be something like: "MeterProvider already shutdown." and for the other one it could be "Mutex for meters is poisoned."